### PR TITLE
[4.1.2] Removed Qiskit Aqua references

### LINKED ISSUE #1331: [4.1.2 Simulating Molecules using VQE] Code example uses Aqua, which is on deprecation path
### Describe the issue

The code example in https://qiskit.org/textbook/ch-applications/vqe-molecules.html still uses qiskit-aqua, which is being deprecated and it is going to be removed from Qiskit soon (maybe in the next release?):

```python
from qiskit.aqua.algorithms import VQE, NumPyEigensolver
...
from qiskit.aqua.components.optimizers import COBYLA, SPSA, SLSQP
from qiskit.aqua.operators import Z2Symmetries
...
from qiskit.aqua import QuantumInstance
...
```

### Describe what you expect to see

Some part of the text also mention Aqua. The code and the related text needs to be adapted to remove all the references to Aqua. The new code lives in terra. In the text, it would be reasonable to just mention "Qiskit" whenever it says "Aqua" or "Qiskit Aqua".

While the full tutorial needs a rewrite, this issue is about the minimal change required to remove Aqua references and leaves out the pedagogic/style changes that could be considering an enhancement.

### PR DESCRIPTION
**Pull request by Ryand1234**
fixes #1331 
# Changes made

removed Qiskit aqua references
# Justification

removed qiskit-aqua references from the code and related text which is being deprecated.

### CODE REVIEW

File: content/ch-applications/vqe-molecules.ipynb [hunk 1]
```diff
@@ -158,7 +158,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 9,
```
Comment (divshacker) [inline content/ch-applications/vqe-molecules.ipynb]: Can you change these execution counts by rerunning all the cells from starting?
Also, you haven't executed some of the last cells. Please rerun them too.

**Review by frankharkins — CHANGES_REQUESTED** [review]
Thank you so much for taking a look at this. While running your notebook I got this error:
```
AttributeError: type object 'Z2Symmetries' has no attribute 'two_qubit_reduction'
```

In this 8th code cell. We need the entire notebook to run correctly before we can merge.

**Comment by Ryand1234** [comment]
@frankharkins I am not getting this error, Its working fine for me. But I am getting 

`QiskitChemistryError: 'PySCF is not installed. See https://sunqm.github.io/pyscf/install.html'` 

error. I tried installing PySCF but its not installing.

**Comment by divshacker** [comment]
> @frankharkins I am not getting this error, Its working fine for me. But I am getting
> 
> `QiskitChemistryError: 'PySCF is not installed. See https://sunqm.github.io/pyscf/install.html'`
> 
> error. I tried installing PySCF but its not installing.

Hey, you are not facing any errors because you haven't updated the Qiskit version. 
Can you show your version info?

**Comment by divshacker** [comment]
Here is mine, Verify if it matches with your system too by running this. 
``` 
import qiskit.tools.jupyter
%qiskit_version_table
```

<h3 style="box-sizing: border-box; font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; font-weight: bold; line-height: 1; color: rgb(0, 0, 0); margin: 0.777em 0px 0px; font-size: 18.004px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Version Information</h3>

Qiskit Software | Version
-- | --
qiskit-terra | 0.19.1
qiskit-aer | 0.10.1
qiskit-ignis | 0.7.0
qiskit-ibmq-provider | 0.18.3
qiskit-aqua | 0.9.5
qiskit | 0.34.0
qiskit-nature | 0.3.0
qiskit-finance | 0.3.0
qiskit-optimization | 0.3.0
qiskit-machine-learning | 0.3.0

**Comment by frankharkins** [comment]
Thanks for your contribution. I'm closing this pull request as this repository is no longer used and is being archived. You can find the latest version of the Qiskit Textbook online at <https://qiskit.org/learn/>, and the new content repository at [Qiskit/textbook](https://github.com/Qiskit/textbook). Please feel free to re-open this pull request there if it's still relevant.

**Pull request closed without merging**